Appz/Appz/Apps/Mail.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  Mail.swift
+//  Appz
+//
+//  Created by Suraj Shirvankar on 12/5/15.
+//  Copyright © 2015 kitz. All rights reserved.
+//
+
+import Foundation
+
+
+public extension AvailableApplications {
+    
+    public struct Mail: ExternalApplication {
+        
+        public let scheme = "mailto:"
+        public let fallbackURL: String? = ""
+    }
+    
+    public func mail(action: Mail.Action) -> Bool {
+        return appCaller.launch(Mail(), action: action)
+    }
+}
+
+// MARK: - Actions
+
+public extension AvailableApplications.Mail {
+    
+    public enum Action {
+        case Email(email: String, subject: String, body: String)
```

Comment:
I thought about optionals but then the api for the end user would be long with unnecessary params.